ea_restaurant_data_loader/lib/entities/app_access_token.ex

```diff
@@ -5,7 +5,8 @@ defmodule EaRestaurantDataLoader.AppAccessToken do
 
   schema "app_access_tokens" do
     field(:token, :string)
-    belongs_to(:refresh_token_id, AppRefreshToken)
+    field(:refresh_token_id, :integer)
+    belongs_to(:refresh_token_id_id, AppRefreshToken, source: :refresh_token_id, define_field: false)
```

Comment:
`_id_id` seems to be a weird name. Could you use a better naming here?

Reply:
Done. it was renamed to `:app_refresh_token_id`